src/Microsoft.DotNet.Wpf/src/PresentationFramework/System/Windows/Controls/VisualStates.cs

```diff
@@ -5,17 +5,14 @@
 
 using System;
 using System.Windows;
-using System.Security;
 using System.Runtime.InteropServices;
 
 using MS.Internal;
 
 namespace System.Windows.Controls
 {
-    /// <summary>
-    /// Names and helpers for visual states in the controls.
-    /// <remarks>THIS IS A SHARED FILE.  PresentationFramework.Design.dll must be rebuilt if changed.</remarks>
-    /// </summary>
+    /// <summary> Names and helpers for visual states in the controls. </summary>
+    /// <remarks> THIS IS A SHARED FILE: PresentationFramework.Design.dll must be rebuilt if changed. </remarks>
```

Comment:
This looks like something, I will have to take a look at.

Reply:
@dipeshmsft There used to be `PresentationFramework.VisualStudio.Design.dll` with old VS (e.g. my 2010 installation has it) as a private assembly (it was providing the design-time support afaik?). But the new ones don't have it.

Reply:
By the way, if you find the remarks irrelevant (which I'm very much confident it is these days), then I'd remove it in this PR before merging it.

Reply:
@dipeshmsft Any news?